apps/backoffice-v2/src/pages/Entities/components/Cases/Cases.Item.tsx

```diff
@@ -92,7 +92,17 @@ export const Item: FunctionComponent<IItemProps> = ({
           </div>
         </div>
         <div className={`ml-auto mr-1 flex -space-x-2 overflow-hidden`}>
-          {!!assignee.id && <UserAvatar fullName={assignee.fullName} avatarUrl={avatarUrl} />}
+          {assignee ? (
+            assignee.avatarUrl ? (
+              <UserAvatar fullName={assignee.fullName} avatarUrl={entityAvatarUrl} />
+            ) : (
+              <Avatar
+                placeholder={createInitials(assignee.fullName)}
+                src=""
+                alt={assignee.fullName}
+              />
+            )
+          ) : null}
```

Comment:
`UserAvatar` covers initial AFAIK. Note this is supposed to be the assignee avatar and not the entity.

```suggestion
    assignee?.avatarUrl && (
        <UserAvatar fullName={assignee.fullName} avatarUrl={entityAvatarUrl} />
    )
```